docs/introduction/get-started/create-account-use-faucet.md

```diff
@@ -134,8 +139,36 @@ State synced to block 179672
       ```shell
       miden consume-notes --account <Account-Id> <Note-Id>
       ```
+5. You should see a confirmation message like this:
 
-  Amazing! You just have created a client-side zero-knowledge proof locally on your machine. 
+      ![Transaction confirmation message](../../img/get-started/transaction-confirmation.png)
+
+6. After confirming you can view the new note status by running the following command:
+
+      ```shell
+      miden notes
+      ```
+
+7. You should see something like this:
+
+      ![Viewing process info](../../img/get-started/processing-note.png)
+
+8. The note is `Processing`. This means that the note the consume transaction was sent but you haven't received confirmation of the consumption yet. You can update your view of the rollup by syncing again:
+
+      ```shell
+      miden sync
+      ```
+
+9. After syncing, you should receive confirmation of the consumed note see something like this after listing the notes:
```

Comment:
There is also a word missing or so?

```
After syncing, you should receive confirmation of the consumed note see something like this after listing the notes:
```

Is it `you should have received`? And here is something off `of the consumed note see something like`

Reply:
Fixed! Yeah it sounded strange after reading it again.